tests/x11regressions/gnomecase/gnome_default_applications.pm

```diff
@@ -61,7 +61,7 @@ sub run() {
     send_key "ctrl-w";                                     #close fileroller
     wait_still_screen;
     assert_and_dclick "gnomecase-defaultapps-htmlfile";    #open html
-    assert_screen 'gnomecase-defaultapps-firefoxopen';
+    assert_screen 'gnomecase-defaultapps-firefoxopen', 60;
```

Comment:
will it work if you just delete the timeout and rely on the timeout? code would be cleaner.